pkg/resources/stream.go

```diff
@@ -109,295 +105,207 @@ func Stream() *schema.Resource {
 	}
 }
 
-type streamID struct {
-	DatabaseName string
-	SchemaName   string
-	StreamName   string
-}
-
-type streamOnObjectID struct {
-	DatabaseName string
-	SchemaName   string
-	Name         string
-}
-
-// String() takes in a streamID object and returns a pipe-delimited string:
-// DatabaseName|SchemaName|StreamName.
-func (si *streamID) String() (string, error) {
-	var buf bytes.Buffer
-	csvWriter := csv.NewWriter(&buf)
-	csvWriter.Comma = streamIDDelimiter
-	dataIdentifiers := [][]string{{si.DatabaseName, si.SchemaName, si.StreamName}}
-	if err := csvWriter.WriteAll(dataIdentifiers); err != nil {
-		return "", err
-	}
-	strStreamID := strings.TrimSpace(buf.String())
-	return strStreamID, nil
-}
-
-// streamIDFromString() takes in a pipe-delimited string: DatabaseName|SchemaName|StreamName
-// and returns a streamID object.
-func streamIDFromString(stringID string) (*streamID, error) {
-	reader := csv.NewReader(strings.NewReader(stringID))
-	reader.Comma = streamIDDelimiter
-	lines, err := reader.ReadAll()
-	if err != nil {
-		return nil, fmt.Errorf("not CSV compatible")
-	}
-
-	if len(lines) != 1 {
-		return nil, fmt.Errorf("1 line at a time")
-	}
-	if len(lines[0]) != 3 {
-		return nil, fmt.Errorf("3 fields allowed")
-	}
-
-	streamResult := &streamID{
-		DatabaseName: lines[0][0],
-		SchemaName:   lines[0][1],
-		StreamName:   lines[0][2],
-	}
-	return streamResult, nil
-}
-
-// streamOnObjectIDFromString() takes in a dot-delimited string: DatabaseName.SchemaName.TableName
-// and returns a streamOnObjectID object.
-func streamOnObjectIDFromString(stringID string) (*streamOnObjectID, error) {
```

Comment:
Shouldn't we keep this method to not introduce breaking (or non-breaking) changes with this change about using the SDK and not changing the way we expect the user to type in identifiers in the config?